mesheryctl/cmd/perf.go

```diff
@@ -82,6 +84,52 @@ func StringWithCharset(length int) string {
 	return string(b)
 }
 
+func AddAuthDetails(req *http.Request, filename string) error {
+	file, err := ioutil.ReadFile(filename)
+	if err != nil {
+		log.Printf("File read failed : %v", err.Error())
+		return err
+	}
+	var tokenObj map[string]string
+	json.Unmarshal(file, &tokenObj)
+	req.AddCookie(&http.Cookie{
+		Name:     tokenName,
+		Value:    tokenObj[tokenName],
+		HttpOnly: true,
+	})
+	req.AddCookie(&http.Cookie{
+		Name:     providerName,
+		Value:    tokenObj[providerName],
+		HttpOnly: true,
+	})
+	return nil
+}
+
+func UpdateAuthDetails(filename string) error {
+	const mesheryAuthToken string = "http://localhost:9081/api/gettoken"
+	req, _ := http.NewRequest("GET", mesheryAuthToken, bytes.NewBuffer([]byte("")))
+	AddAuthDetails(req, filename)
+
+	client := &http.Client{}
+	resp, _ := client.Do(req)
+	defer resp.Body.Close()
+
+	data, _ := ioutil.ReadAll(resp.Body)
+	log.Printf("%v", string(data))
+	ioutil.WriteFile(filename, data, os.ModePerm)
+	return nil
+}
+
+func defaultAuthConfig() string {
+	// Find home directory.
+	home, err := os.UserHomeDir()
```

Comment:
I wonder if we can reuse this function if we have it return the location - [setFileLocation](https://github.com/layer5io/meshery/blob/master/mesheryctl/cmd/vars.go#L41) or if we create a getFileLocation.